coral/docs/development-with-remote-api.md

```diff
@@ -0,0 +1,30 @@
+# Development with remote API
```

Comment:
Maybe we can extend the documentation with:
- short explanation why you want to use a remote API and what is happening if you don't do that (we use mocked data etc.)
- note that `.env.remote-api` and also `.cert/localhost.crt` need to be located in `klaw/coral` (I'm not sure that is super obvious to people right away)
- note that `[KLAW API ORIGIN]` should be replaced with the target the people want to use
- add commands or links on how to add the cert to the locally trusted roots

Reply:
Pushed fixes to all comments except the last one. I think it is fair to assume other contributors to either know how to add certificate to trusted roots, or then how to find the instructions. I was not able to find _official_ references that I would be happy to include in the documentation. For some reason I am a bit cautious referring to external blogposts.